Sources/carton/Server/Server.swift

```diff
@@ -64,8 +94,31 @@ final class Server {
       mainWasmPath: builder.mainWasmPath,
       customIndexContent: customIndexContent,
       package: package,
-      onWebSocketOpen: { [weak self] in
-        self?.connections.insert($0)
+      onWebSocketOpen: { [weak self] ws, environment in
+        ws.onText { _, text in
+          guard
+            let data = text.data(using: .utf8),
+            let event = try? self?.decoder.decode(Event.self, from: data)
+          else {
+            return
+          }
+
+          switch event {
+          case let .stackTrace(rawStackTrace):
+            guard environment == .firefox else { break }
+
+            let stackTrace = rawStackTrace.firefoxStackTrace
+
+            terminal.write("\nAn error occurred, here's a stack trace for it:\n", inColor: .red)
+            stackTrace.forEach { item in
+              terminal.write("  \(item.symbol)", inColor: .cyan)
+              terminal.write(" at \(item.location)\n", inColor: .grey)
```

Comment:
Could we consider a color other than grey? Not sure if the problem is in my terminal settings, but:
<img width="910" alt="Screen Shot 2020-11-26 at 17 14 34" src="https://user-images.githubusercontent.com/2340723/100331391-0f09b500-300b-11eb-837c-7dc65c27c2a6.png">

Reply:
Thanks for testing it, I changed it to use the default color